src/ovh-ui-angular/controllers/datagrid.controller.js

```diff
@@ -95,6 +96,19 @@ export default class {
             .then(row => this.loadRandom(row));
     }
 
+    loadGroupedData () {
+        const originalData = angular.copy(data);
+        const groupedData = [];
+        while (originalData.length > 0) {
+            const parentRow = groupedData[groupedData.push(originalData.pop()) - 1];
+            parentRow.subrows = [];
+            for (let i = 0; i < 4 && originalData.length > 0; i++) { // eslint-disable-line
```

Comment:
(optional) Why `// eslint-disable-line`, for the magic number ?
Even if it's for the documentation, we should avoid eslint-disable :)